portal/src/texts/blog/kulturForDesignsystem/kulturForDesignsystem.mdx

```diff
@@ -0,0 +1,60 @@
+Etter mye lesing, mange flere inspirerende samtaler og diskusjoner, forsøkte vi å overbevise andre utviklere og designere i SpareBank 1 
+om at vi kunne jobbe på en smartere måte til stor fordel for både brukerne og oss. Etter hvert begynte andre i industrien å kalle disse 
+prinsippene for “designsystem” og vi oppdaget plutselig en enda større gruppe likesinnede som jobbet med lignende ideer. 
+Vi hadde etter hvert fått nok moment til å kunne lage et designsystem som flere og flere team i SpareBank 1 både kunne bruke og ønsket å bruke.
+
+I stedet for å starte på ideen og spre den til teamene, kunne vi gjennomført en problemanalyse, forsøkt å få støtte fra ledelsen, 
+formet et mandat og sparket i gang et prosjekt som skulle formalisere en ny design- og utviklingsmetode og tilhørende verktøykasse som alle skulle bruke. 
+Det ville kanskje fungert til en viss grad. Men hvis man ønsker at noe så stort skal virkelig være levende og intuitivt, 
+må det påvirkes kontinuerlig av alle menneskene som skal etterleve det. Den kollektive kunnskapen må på en selvdreven måte kunne gjennomsyre 
+både prosess og produkt. Den typen oppførsel er vanskelig å regulere med mandat og prosjekt. Den kommer gjennom kultur og praktiske opplevelser.
+
+## Fremtind og komponentballen
+Denne læringen tok vi med oss når Fremtind ble formet gjennom en fusjon av SpareBank 1 Forsikring og DNB Forsikring i 2019. 
+Det var derfor innlysende, både for utviklerne, designerne og ledelsen, at vi trengte et designsystem. Men vi trengte også en gruppe med evangelister 
+som skulle sørge for felles innsats til å skape et system som alle både kunne bruke og ønsket å bruke. Et eget designsystem-team ble etablert fra starten.
+
+Alle de eksisterende webløsningene i Fremtind var arvet fra SpareBank 1 Forsikring og hadde derfor kun SpareBank 1-profil. 
+Plutselig skulle vi støtte kunder i både SpareBank 1- og DNB-bankene. En ny webløsning for våre rådgivere var blant flere allerede planlagte 
+oppgaver før fusjonen og denne skulle nå designes og utvikles i det nye selskapet. Vi trengte derfor raskt å fordøye det vi kunne av den nye Fremtind-profilen 
+og lage komponenter som teamene skulle bruke i det nye selskapet. Vi måtte få komponententballen til å rulle i vårt nye designsystem.
+
+Ballen begynte å rulle fort! Mange hadde allerede opplevd fordelene av å jobbe der det fantes veletablerte designsystem, 
+så visjonen var allerede godt forstått og ble raskt til skisser og kode som igjen ble tatt i bruk fortløpende i nye løsninger. 
+Etter hvert som flere begynte å bruke designsystemet, tenke vi at det ville være nødvendig med interne designsystemkurs og planer ble lagt for dette. 
+Disse har aldri blitt realisert. Vi har aldri hatt formelle designsystemkurs i Fremtind, men folk har allikevel lært hvordan det fungerer og 
+resultatet av arbeidet har blitt anerkjent med [designpris fra Visuelt](https://www.grafill.no/visuelt/vinnere/2021/digital-design/designsystemer/designsystem-jokul). 
+
```

Comment:
Ingen designsystemkurs, men kanskje si noe om designsystemforum? Ukentlig og frivilling - og stort engasjement og oppmøte?

Reply:
Skikkelig enig. Tatt med et par setninger nå. Si ifra om den ble litt for svulstig. :)

Reply:
Savner noe om hva designsystem-teamet egentlig er eller gjør. `Et eget designsystem-team med et mandat om å sørge for bred deltakelse i designsystemet, og spre kulturen for et designsystem i organisasjonen` eller noe sånt?

Her ønsker jeg meg også en illustrasjon av mantel-teorien.